docs/guides/docker.md

```diff
@@ -8,224 +8,216 @@ author:
     container_name: sonarr
+    ports:
+      - 8989:8989
     environment:
       - PUID=1000
       - PGID=1000
-      - TZ=Europe/London
-    ports:
-      - 8989:8989
+      - TZ=Europe/London # Change this based on your timezone
     volumes:
       - './sonarr:/config'
       - '/home/user/data:/data'
-    networks:
-      - guide
     restart: unless-stopped
   radarr:
     image: lscr.io/linuxserver/radarr:latest
     container_name: radarr
+    ports:
+      - 7878:7878
     environment:
       - PUID=1000
       - PGID=1000
-      - TZ=Europe/London
-    ports:
-      - 7878:7878
+      - TZ=Europe/London # Change this based on your timezone
     volumes:
       - './radarr:/config'
       - '/home/user/data:/data'
-    ports:
-      - 7878:7878
-    networks:
-      - guide
     restart: unless-stopped
-  jackett:
-    image: lscr.io/linuxserver/jackett:latest
-    container_name: jackett
+  prowlarr:
+    image: lscr.io/linuxserver/prowlarr:latest
+    container_name: prowlarr
+    ports:
+      - 9696:9696
     environment:
       - PUID=1000
       - PGID=1000
-      - TZ=Europe/London
-      - AUTO_UPDATE=true # Optional
-      - RUN_OPTS= # Optional
-    ports:
-      - 9117:9117
+      - TZ=Europe/London # Change this based on your timezone
     volumes:
-      - './jackett:/config'
+      - './prowlarr:/config'
       - '/home/user/data:/data'
-    networks:
-      - guide
     restart: unless-stopped
-networks:
-  guide:
-    external: true
 ```
+==-
+
+We would also recommend sorting your media-related folders in the following way for ease of use:
+- `/home/user/data/torrents/` - where you torrent client will save files.
+- `/home/user/data/usenet` - where your usenet client will save files.
+- `/home/user/data/media` - where your hardlinked files will reside, so that they can be read by your media server (Plex/Jellyfin/etc.)
+  - `/home/user/data/media/tv` - where Sonarr will hardlink files.
+  - `/home/user/data/media/movies` - where Radarr will hardlink files.
+  - You can take this concept further and make more subfolders in `media` if you'd like to have one more than one Sonarr or Radarr instance. People
+  sometimes do this if they'd like both a 4K and an HD version of a piece of media, as both the *arrs can only hardlink one version at a time.
 
-After all of that you'll want to go into `/home/user/projects/automation` and run `docker compose up -d`. This will deploy all of the containers
-in a detached mode, but since it's your first time running them, Docker will pull the images for each of these containers.
+Paths for Docker are extremely important as configuring them incorrectly can break hardlinking and cause you to waste space!
 
-Once all of that's done, you'll want to run `docker compose down`, which will shut down each of the containers.
+After all of that you'll want to go into `/home/user/projects/automation` and run `docker compose up -d`.  This will deploy all of the containers
```

Comment:
The user may end up with `docker-compose` or `docker compose` depending on their distribution and how old it is. This *may* not be the exact command to do this. It might be worth teaching the user how to install the latest compose from GitHub if we want them to have a consistent experience. At the very least, this probably deserves a footnote at the first possible mention of compose.